cmd/kustomizepatch/vault.go

```diff
@@ -0,0 +1,99 @@
+package kustomizepatch
```

Comment:
This file is an exact copy of `cmd/generate/vault.go`

Reply:
Can we move these different files to a common package? Or would we just remove the generate command?

Reply:
Definitely move to a common package. I wanted to minimize time spent on refactoring, while still proving it works so I don't miss any possible obstacles.

`konfigure generate` would still be useful to get the output on personal machines with fewer restrictions as to directory placement/flags used.

Reply:
This is still outstanding, I will definitely do it, but let's get some eyes on what I have done since last review.